app.js

```diff
@@ -9,12 +9,16 @@ const app = express();
 const helmet = require("helmet");
 app.use(compression());
 
-
-// loggin
+// logging
+morgan.token("locals", (req, res) => {
```

Comment:
Fancy!

Reply:
I guess this automatically checks for `undefined`

Reply:
Yes. `undefined` becomes `-`